docs/overview/changelog.md

```diff
@@ -1,15 +1,13 @@
 # Version history
 
-This page the release history for the Zulip server. See also the
-[Zulip release lifecycle](../overview/release-lifecycle.md).
+This page contains the release history for the Zulip 6.x stable
+release series. See the [current Zulip changelog][latest-changelog]
+for newer release series, or the [commit log][commit-log] for an
+up-to-date list of raw changes.
```

Comment:
Can we drop "raw"?

Reply:
I guess the list is raw, not the changes?

Reply:
Changed to "all changes".